lib/pure/strutils.nim

```diff
@@ -84,31 +84,42 @@ from std/private/strimpl import cmpIgnoreStyleImpl, cmpIgnoreCaseImpl, startsWit
 
 
 const
-  Whitespace* = {' ', '\t', '\v', '\r', '\l', '\f'}
+  Whitespace* = {' ', '\t', '\v', '\r', '\n', '\f'}
     ## All the characters that count as whitespace (space, tab, vertical tab,
     ## carriage return, new line, form feed).
 
   Letters* = {'A'..'Z', 'a'..'z'}
     ## The set of letters.
 
+  Vowels* = {'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'}
+    ## The set of vowels.
+  
```

Comment:
in future work, I'd also like these sets:
* `ControlChars` (with semantics derived from `iscntrl`)
* `PrintableChars` (with semantics derived from `isprint`)

Reply:
I will definitely look into that!

Reply:
I'm not sure how I should do that, and what exactly I should do... so I could probably do that in future PR.

Reply:
in that future PR, you could iterate over all chars, and call iscntrl, isprint on it, and populate a set with the values from it:
`const ControlChars = {'\12', ...}`
  # computed it offline and verify it in tests via tests/stdlib/tstrutils.nim

```nim
# tstrutils.nim
proc iscntrl(c: cint): cint {.importc.}
block: # ControlChars
  for i in char.low .. char.high:
    doAssert iscntrl(i.cint) > 0 = i in ControlChars
```